### DISCUSSION TITLE
Create Campaign Not working

### DISCUSSION THREAD

**Discussion by lifelonglearner127** [denisneuf/python-amazon-ad-api#98] [Announcements]
**Cannot create a campaign**
I can successfully get campaigns, but unfortunately cannot create campaigns. 
It says `create_campaigns() takes 1 positional argument but 2 were given`

**To Reproduce**
```
    credentials = {
        "client_secret": "***",
        "client_id": "***",
        "refresh_token": "***",
        "profile_id": "****",
    }
    sp = sponsored_products.Campaigns(credentials=credentials)
    ret = sp.list_campaigns()
    ret = sp.create_campaigns(
        []
    )
    print(ret)
```

**Environment**
 - OS: Ubuntu 22.04
 - Poetry
 - python-amazon-ad-api [0.3.4]

**Comment by denisneuf** [comment]
You need pass a body campaign please check the help:[
](https://python-amazon-ad-api.readthedocs.io/en/latest/sp/campaigns.html#ad_api.api.sp.Campaigns.Campaigns.create_campaigns)

not sure in your code why you use same ret to get a list of campaigns an d inmideately use the same to create with an empty []

**Comment by denisneuf** [comment]
https://python-amazon-ad-api.readthedocs.io/en/latest/sp/campaigns.html#ad_api.api.sp.Campaigns.Campaigns.create_campaigns

**Reply by lifelonglearner127** [reply]
thanks, the issue was because I passed the campaign data as positional argument.
If I pass it as a named argument, it works.
